src/main/java/mate/academy/mybookshop/entity/BookEntity.java

```diff
@@ -5,11 +5,17 @@
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
+import jakarta.persistence.Table;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
 
 @Entity
 @Data
+@SQLDelete(sql = "UPDATE books SET is_deleted = true WHERE id=?")
+@SQLRestriction(value = "is_deleted=false")
+@Table(name = "books")
 public class BookEntity {
```

Comment:
```suggestion
public class Book {
```

Reply:
I think it's ok for naming BookEntity like BookRepository, or BookService

Reply:
I wouldn't say that this is a big deal, we have `...Entity` naming in our project ourselves, but I lean more toward what Andrii suggested